### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by MrClon**
Tested only with WorkTab v0.17.1.3 and RimWorld v0.17.1557 rev1148, but i think in this case it OK

### CODE REVIEW

File: Languages/Russian/Keyed/Keyed-Russian.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <WorkTab.PlaySoundsTip>Воспроизводить звук при изменении приоритета?</WorkTab.PlaySoundsTip>
+    <WorkTab.PlayCrunch>Хруст</WorkTab.PlayCrunch>
+    <WorkTab.PlayCrunchTip>Воспроизводить хрустящий звук когда колонисту назначена работа в которой он не умел</WorkTab.PlayCrunchTip>
+
+    <!-- scheduler -->
+    <WorkTab.SchedulerHourTip>Временной интервал {0}-{1} сейчас {2}\n\n{3}</WorkTab.SchedulerHourTip>
+    <WorkTab.Selected>выделен</WorkTab.Selected>
+    <WorkTab.NotSelected>не выделен</WorkTab.NotSelected>
+    <WorkTab.ClickToSelect>Клик выделит\nShift+клик выделит только этот интервал</WorkTab.ClickToSelect>
+    <WorkTab.ClickToFocus>Click to focus</WorkTab.ClickToFocus>
```
Comment (FluffierThanThou) [inline Languages/Russian/Keyed/Keyed-Russian.xml]: you missed a line ;)

Comment (MrClon) [reply Languages/Russian/Keyed/Keyed-Russian.xml]: Didn't find this line in interface, so I can't choose correct translation

Comment (FluffierThanThou) [reply Languages/Russian/Keyed/Keyed-Russian.xml]: hmm it's entirely possible that it's no longer used. Thanks a bunch anyway, it's included in the A18 (pre)release

Comment (FluffierThanThou) [reply Languages/Russian/Keyed/Keyed-Russian.xml]: actually, it _is_ used. It's displayed in the tooltip when hovering over an hour in the time bar.

**Pull request merged**